topostats/validation.py

```diff
@@ -187,6 +187,14 @@ def validate_config(config: dict, schema: Schema, config_type: str) -> None:
                 "lower_norm_bound": float,
             },
             "vetting": {
+                "class_conversion_size_thresholds": Or(
+                    None,
+                    # List of lists of 3 integers and 2 integers
+                    [[[int, int, int], [int, int]]],
+                    error="Invalid value in config for 'grainstats.vetting.class_conversion_size_thresholds', this"
```

Comment:
Without a space at the end of the line the concatenation of strings will end up reading `thisneeds`...

```suggestion
                    error="Invalid value in config for 'grainstats.vetting.class_conversion_size_thresholds', this "
```